sdk/src/loom2d/Display/DisplayObject.ls

```diff
@@ -322,6 +323,8 @@ package loom2d.display
             while (ancestor != this && ancestor != null)
                 ancestor = ancestor.parent;
 
+            _cachedStage = null;
```

Comment:
Sorry, I meant the code that I added.. also I'm not sure if it's possible to hardcode it, since it's supposed to return null if it's not on the stage and you have to traverse anyway. We already do have Loom2D.stage too.